modules/checking-mco-status.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="checking-mco-status_{context}"]
+= Checking MCO status
+
+To see the MachineConfigs assigned to your {product-title} nodes and see their status, use the `oc` command:
+
+.Procedure
+. To see the number of machines available on your cluster and the role that each plays, type:
++
+[source,terminal]
+----
+$ oc get machineconfigpool
+NAME      CONFIG                  UPDATED  UPDATING   DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT DEGRADEDMACHINECOUNT  AGE
+master    rendered-master-dd…     True     False      False     3             3                  3                                0                     4h42m
+worker    rendered-worker-fde…    True     False      False     3             3                  3                                0                     4h42m
+----
++
+Here there are three master and three worker nodes. All machines are updated
+and none are currently updating. Because none are degraded (False), you can
```

Comment:
All machines are updated and none are updating is redundant.

Say something this instead:
"Since all nodes are Updated and Ready and none are Degraded, we can tell that there are no issues..."